Copy edits for btcli permissions guide. (#541)
opentensor/developer-docs: docs/btcli-permissions.md

The workstations you use to do this work can be referred to as a permissionless workstation (requiring neither private key), a coldkey workstation or a hotkey workstation, depending on which private key is provisioned.

> 1. A **permissionless workstation** has only coldkey *public keys* on it. These are sufficient for viewing all public information about a wallet, such as TAO and alpha stake balances. That way, information about subnets, miners, and validators can be viewed without introducing the security risk of initializing your private keys on a device.

Reviewer: ```suggestion
1. A **permissionless workstation** has only coldkey *public keys* on it. Public keys are sufficient for viewing all information about a wallet, such as TAO and alpha stake balances. Information about wallets, subnets, miners, and validators can and should be viewed without initializing your private keys on a device, to avoid the security risk of compromising your keys.
```

    :::tip coldkey workstation security
    See [Permissionless workstation](./getting-started/coldkey-hotkey-security#permissionless-workstation)
    :::

1. A **coldkey workstation** contains a coldkey in the `wallet_path`. For any coldkey associated with mainnet TAO, the coldkey workstation should be held to the highest possible security standards. 

    :::tip coldkey workstation security
    See [Coldkey workstation](./  getting-started/coldkey-hotkey-security#coldkey-workstation)
    :::

> 1. **A hotkey workstation**—which is generally a server used for mining or validation—contains a hotkey private key in the `wallet_path` located in the `btcli config`, as well as a coldkey public key for the corresponding coldkey. Compromised hotkeys can damage your reputation if they are used to maliciously to submit false weights as a validator, or bad work as a miner. However, ownership of TAO or alpha stake can only be transferred with a coldkey, and a leaked hotkey can be swapped using the coldkey; therefore hotkey leaks are far less dangerous than coldkey leaks.

Reviewer: ```suggestion
1. **A hotkey workstation**&mdash;generally mining or validation server&mdash;contains a hotkey private key in the `wallet_path` located in the `btcli config`, as well as a coldkey public key for the corresponding coldkey. Compromised hotkeys can damage your reputation if they are used to maliciously to submit inaccurate weights as a validator, or bad work as a miner. However, ownership of TAO or alpha stake can only be transferred with a coldkey, and a leaked hotkey can be swapped using the coldkey; therefore hotkey leaks are far less dangerous than coldkey leaks.
```

Author: You can make a renderable em-dash with shift option dash. It rendered as a full emdash on my local build FWIW. You just can't tell in git. Would you prefer I use $mdash so that it is obvious in git?

### `wallet`

`wallet` subcommands have a variety of uses and must be run on all different kinds of workstation.

> The `walleet` command is required to provision keys to `btcli`, so btcli can access your wallet. This is essentially the equivalent of logging in/authentication. This is true for both coldkeys, which all users require, and hotkeys, which are required only by miners and validators as well as for advanced functions.

Reviewer: ```suggestion
The `wallet` command is required to provision keys to `btcli`, so it can access your wallet. This is essentially the equivalent of logging in/authentication. This is true for both coldkeys, which all users require, and hotkeys, which are required only by miners and validators as well as for advanced functions.
```

1. **`new coldkey`** is used to initialize a coldkey workstation using a newly created *seed phrase*. This is a high security risk operation due to the inherent risk of handling the seed phrase.

1. **`regen coldkey`** is used to initialize a coldkey workstation using a pre-existing wallet's *seed phrase*. This is a high security risk operation due to the inherent risk of handling the seed phrase.

1. **`new hotkey`** is used to initialize a hotkey workstation using a newly created *seed phrase*. This is a high security risk operation due to the inherent risk of handling the seed phrase. Hotkeys should be created on secure coldkey workstation and then carefully provisioned to working nodes for mining and validation.

1. **`regen hotkey`** is used to initialize a hotkey workstation using a pre-existing wallet's *seed phrase*. This is a high security risk operation due to the inherent risk of handling the seed phrase.

#### Permissionless operations 

- **`btcli wallet balance`**: Displays a wallet balance.
- **`btcli wallet overview`**: Displays a wallet overview.

#### Operations requiring coldkey private key:

> 

Reviewer: the bullet formatting across this page should be made consistent. I think probably they should be capitalized if they have a period at the end. If it's colon-separated the term being defined isn't part of the sentence.

Author: I was thinking with them being lowercase, you an read them straightforwardly as sentences:
`btcli wallet balance` displays wallet balance. That was at least my intention. I can go back and make them all Uppercase. I see your point about the colon breaking up the sentence, but then I ask the question, do we need a colon at all? Couldn't we just make proper sentences by getting rid of the colons and using lowercase?

Reviewer: i guess i don't really care, just as long as it's readable and reasonably consistent

Author: K I went with no colon to make complete sentences.

### `utils`

#### `btcli utils convert`

> `btcli utils convert` is a convenience command for performing conversions between minimal units (RAO) and TAO, or other chain-specific conversions. It is permissionless (no key required) because it performs no on-chain operation, just a local calculation. 

Author: Wait, is RAO still a thing after Dynamic TAO?

Reviewer: Rao is also the name of the currency denomination 1 billionth of a TAO
https://docs.bittensor.com/glossary#rao
